modules/ossm-migrating-multitenant-without-cert-manager.adoc

```diff
@@ -0,0 +1,187 @@
+----
++
+<1> The `spec.namespace` field in your `Istio` resource must be the **same** namespace as your `ServiceMeshControlPlane`. If you set the `spec.namespace` field in your `Istio` resource to a different namespace than your `ServiceMeshControlPlane`, the migration will not work properly.
+<2> controlplanes by default watch the entire cluster and when managing multiple controlplanes on a single cluster, you must narrow the scope of each controlplane by setting `discoverySelectors`. In this example, we use the label `tenant` but you can use any label or combination of labels that you choose.
+<3> If you are migrating metrics and tracing, uncomment and update the `extensionProviders` fields according to your tracing and metrics configurations.
+
+. Add your `tenant` label to each one of your dataplane namespaces by running the following command:
++
+[source,terminal]
+----
+$ oc label ns bookinfo tenant=tenant-a
+----
++
+[NOTE]
+====
+With {SMProduct} 2.6, namespaces were enrolled into the mesh by adding them to the `Service MeshMemberRoll` resource. In {SMProduct} 3, you must label each one of your dataplane namespaces with the `discoverySelectors.matchLabels` label.
+====
+
+=== Migrating workloads
+
+Now you can migrate your workloads from the {SMProduct} 2.6 controlplane to the {SMproduct} 3.0 controlplane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately.
+====
+
+. Find the current `IstioRevision` for your {SMProduct} 3.0 controlplane by running the following command:
++
+[source,terminal]
+----
+$ oc get istios istio-tenant-a
+NAME             REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+istio-tenant-a   1           1       0        istio-tenant-a    Healthy   v1.24.1   30s
+----
++
+[NOTE]
+====
+The naming format of your revisions depends on which upgrade strategy you choose for your `Istio` instance.
+====
+
+. Copy the `ACTIVE REVISION` to use as your `istio.io/rev` label in the next step.
+
+. Update injection labels on the `dataplane` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label ns bookinfo istio.io/rev=istio-tenant-a maistra.io/ignore-namespace="true" --overwrite=true
+----
++
+This adds the following labels to the namespace:
++
+.. The `istio.io/rev: istio-tenant-a` label: Ensures that any new pods that get created in that namespace connect to the {SMProduct} 3.0 proxy.
++
+.. The `maistra.io/ignore-namespace: "true"` label: Disables sidecar injection for {SMProduct} 2.6 proxies in the namespace so that {SMProduct} 2.6 stops injecting proxies in this namespace, and any new proxies are injected by the {SMProduct} 3.0 `controlplane`. Without this label, the {SMProduct} 2.6 injection webhook tries to inject the pod and it will connect to the {SMProduct} 2.6 proxy as well as refuse to start. This happens because the injection webhook has the {SMProduct} 2.6 Carrier Infrastructure Network (CNI) annotation.
```

Comment:
```suggestion
.. The `maistra.io/ignore-namespace: "true"` label: Disables sidecar injection for {SMProduct} 2.6 proxies in the namespace so {SMProduct} 2.6 stops injecting proxies in this namespace, and any new proxies are injected by {SMProduct} 3.0. Without this label, the {SMProduct} 2.6 injection webhook tries to inject the pod and the injected sidecar proxy will refuse to start since it will have both the {SMProduct} 2.6 and the {SMProduct} 3.0 Container Network Interface(CNI) annotations.
```